<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>10</y>
      <width>351</width>
      <height>630</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Tab1">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="x">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="X_value">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="y">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Y_value"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="z">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="Z_value"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ctan angle( X, Y )</string>
      </property>
     </widget>
     <widget class="QComboBox" name="PortChoice">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="openPort">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closePort">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QListWidget" name="angle_xy">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="angle_xz">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="angle_yz">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ctan angle ( X, Z )</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ctan angle ( Y, Z )</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="TurnOnEngine">
         <property name="text">
          <string>Turn on</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TurnOffEngine">
         <property name="text">
          <string>Turn off</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ENGINES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="english">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>550</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>english</string>
      </property>
     </widget>
     <widget class="QPushButton" name="german">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>550</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>german</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab2">
     <attribute name="title">
      <string>Graphs</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>331</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QChartView" name="chartView_x" native="true"/>
       </item>
       <item>
        <widget class="QChartView" name="chartView_y" native="true"/>
       </item>
       <item>
        <widget class="QChartView" name="chartView_z" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="connection">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
